utils/utils.py

```diff
@@ -110,10 +110,14 @@ def sanity_check(args: Union[argparse.Namespace, DictConfig]):
     assert args.file_with_video_paths or args.video_paths, '`video_paths` or `file_with_video_paths` must be specified'
     assert os.path.relpath(args.output_path) != os.path.relpath(args.tmp_path), 'The same path for out & tmp'
     if args.show_pred:
-        print('You want to see predictions. So, I will use only the first GPU from the list you specified.')
-        args.device_ids = [args.device_ids[0]]
+        if args.cpu is False:
+            print('You want to see predictions. So, I will use only the first GPU from the list you specified.')
+            args.device_ids = [args.device_ids[0]]
         if args.feature_type == 'vggish':
             print('Showing class predictions is not implemented for VGGish')
+        if args.feature_type == 'clip':
+            if args.pred_texts is None:
```

Comment:
TODO to myself: if I will implement zero-shot on Kinetics 400, I need to change this.